vulture_os/toolkit/api_parser/cisco_meraki/cisco_meraki.py

```diff
@@ -145,20 +156,44 @@ def execute(self):
                         nb_events = len(tmp_logs['events'])
                         logs = tmp_logs['events']
 
-                        def format_log(log):
+                        def format_network_log(log):
+                            log['log_type'] = "network"
                             log['organization_id'] = orga['id']
                             log['organization_name'] = orga['name']
                             log['network'] = network['name']
                             log['product'] = product_type
                             log['timestamp'] = log['occurredAt']
                             return json.dumps(log)
 
-                        self.write_to_file([format_log(l) for l in logs])
+                        self.write_to_file([format_network_log(l) for l in logs])
                         # Writting may take some while, so refresh token in Redis
                         self.update_lock()
 
                         if nb_events > 0:
                             # No need to make_aware, date already contains timezone
                             self.frontend.cisco_meraki_timestamp[f"{network['id']}_{product_type}"] = tmp_logs['pageEndAt']
 
+
+            logger.info(f"[{__parser__}]:execute: Getting organisation security events", extra={'frontend': str(self.frontend)})
+
+            since = self.frontend.cisco_meraki_timestamp.get("security_events", (timezone.now()-timedelta(days=1)).isoformat())
+            security_events = self.get_organization_appliance_security_events(since, orga['id'])
```

Comment:
big problem here: the timestamp is not specified **per**-organisation!
The same problem exists for other types of logs! (see line 142)